### PR TITLE
improve delete/modify merge conflict section (resolve git merge conflicts page)

### PR DESCRIPTION
**Pull request by alisonjo315**
("to manually _resolve_ delete merge conflicts") and other improvements to delete/modify merge conflicts intro/explanation text

**Note:** Please fill out the PR Template to ensure proper processing.

Closes: #6048 

## Summary

**[Resolve Git Merge Conflicts](https://pantheon.io/docs/git-resolve-merge-conflicts)** - Improve delete/modify merge conflict section.

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alisonjo315** [comment]
The one other thing I was thinking is to mention the "auto-resolve merge conflicts" option, in the [Resolve Conflicts from Multidevs](https://pantheon.io/docs/git-resolve-merge-conflicts#resolve-conflicts-from-multidevs) section of the same page.

image

IDK what y'all think about that, but, just putting it out there for consideration (if supported, I'll add another commit to this PR, or a new one if you prefer).

**Comment by alisonjo315** [comment]
I'm participating in Hacktoberfest, and I'd love if you'd consider adding `hacktoberfest-accepted` as a label to this PR, so I can get credit for this contribution (if you agree).  You don't necessarily have to approve/merge the PR to apply this label, it would just mean that you view this PR as valid/not spam/genuinely helpful (in theory) -- that sort of thing.

* [Examples of PRs considered "low quality"](https://hacktoberfest.digitalocean.com/details#quality) under Hacktoberfest values/principles.  (List of Hacktoberfest values is at the top of that same page.)
* [Explanation of where the `hacktoberfest-accepted` label comes in](https://hacktoberfest.digitalocean.com/details#details).

Thank you!

File: source/content/git-resolve-merge-conflicts.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ How you resolve a merge conflict depends on what type of conflict you're faced w
 
 ### Resolve delete/modify Conflicts
 
-To manually delete merge conflicts from the terminal, use the following commands in sequence. Start by identifying the file that is generating a delete error.
+When one commit removes a file and another commit modifies that file, a merge conflict is created.  To resolve such a delete/modify merge conflict from the terminal, use the following commands in sequence. Start by identifying the file that is generating a delete error.
```
Comment (alexfornuto) [inline source/content/git-resolve-merge-conflicts.md:75]: For copy edit: switch to active voice.

Comment (alisonjo315) [reply source/content/git-resolve-merge-conflicts.md:75]: @alexfornuto Thanks for the review!  Could you give an example of the kind of phrasing you mean?  I attempted to mimic other examples on this page, but I may have misunderstood.  (I do know passive vs. active voice in theory, I'm just not sure in this specific situation.)  Thank you!

Comment (alexfornuto) [reply source/content/git-resolve-merge-conflicts.md:75]: @alisonjo315 Sorry for the confusion, I added that while we were bug grooming as a note for ourselves. We'll handle the copy editing from here. Thanks!

Comment (alisonjo315) [reply source/content/git-resolve-merge-conflicts.md:75]: Ahhhh ok, gotcha, cool beans 😎

Comment (alexfornuto) [reply source/content/git-resolve-merge-conflicts.md:75]: Indeed, you did follow the existing phrasing on this page, which is also passive. I hope you don't mind if I push commits up to this fork/branch covering more than this section of the page.

Comment (alisonjo315) [reply source/content/git-resolve-merge-conflicts.md:75]: Oh of course not, by all means.  I appreciate the info, and I look forward to seeing how it turns out :)

**Review by alexfornuto — APPROVED** [review]
:+1:

**Pull request merged**